src/scripts/countMovies.js

```diff
@@ -0,0 +1,3 @@
+export default function countMovies(tvArrayCount) {
```

Comment:
I would suggest you try to use arrow functions for the following best practices on JavaScript!